packages/encoding/src/data/index.ts

```diff
@@ -6,9 +6,10 @@ run()
 async function run() {
   const data = await getOnChainData(
     '0x46c212912be05a090a9300cf87fd9434b8e8bbca15878d070ba83375a5dbaebd'
+    // '0x5a81a54eb926a240264f9a481810242987ae566e82f29bcdd8cd32063e1976c5'
```

Comment:
Why is it commented out?

Reply:
The first hash allows you to get the original data we had, the second is for getting data with forced actions